plugins/services/src/js/components/forms/ContainerServiceFormSection.js

```diff
@@ -191,9 +205,14 @@ class ContainerServiceFormSection extends Component {
 
           <FormGroup
             className="column-3"
-            required={true}
```

Comment:
These were used `FormGroup` at some point. It would pass the `required` field to its children, so both the label would display the red asterisk and the fields would get the `required` property.
Why was this removed? https://github.com/dcos/dcos-ui/commit/58629aa5395666d36e45b4a3f5cb75a551cbafa6#diff-90c826309cb4746e318b12cb47b78ec6L15